File: w6/pset6/mario.py
from cs50 import get_int

# input Higth and ceck height number range 1 to 8
while True:
    height = get_int("Height: ")
    if height > 0 and height <= 8:
        break


# Create mario
for i in range(height):
    print(" " * (height - (i + 1)), end="")
    print("#" * (i + 1), end="")
    print("  " + "#" * (i + 1))

File: w1/mario.c
#include <stdio.h>
#include <cs50.h>

int getInt(string text);


int main(void)
{
    int n = getInt("Hight: ");

    // new line for loop
    for (int i = 0; i < n; i++)
    {
        // space
        for (int s = 1 ; s < n - i; s++)
        {
            printf(" ");
        }

        // # : new letter for loop
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }

        printf("  "); // space

        // #: letter loop 2
        for (int k = 0; k <= i; k++)
        {
            printf("#");
        }

        printf("\n");

    }

    return 0;
}

// function check input and get input
int getInt(string text)
{
    int number = 0;
    do
    {
        number = get_int("%s", text);
    }
    while (number <= 0 || number > 8);

    return number;
}